tools/cli/internal/cli/merge/merge_test.go

```diff
@@ -12,15 +12,52 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build unit
```

Comment:
if we don't plan to have int or e2e I think we can skip this for now

Reply:
isn't this tag also used to exclude files from the final build when running `go build`?

Reply:
no, tests are excluded by default by the compiler, this was a "clever hack" I did for the CLI to be able to run different tests with different commands, nothing less nothing more